client/maps/map-details.jsx

```diff
@@ -127,13 +138,24 @@ class NameForm extends React.Component {
   }
 }
 
-@form({ description: required('Enter a map description') })
+@withTranslation(TranslationNamespace.Global, { withRef: true })
```

Comment:
Remove `withTranslation` decorators on forms and fix the `@form` decorator call to `withTranslation` to pass the explicit namespace